CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+<a name="5.0.0"></a>
+# 5.0.0 (Unreleased)
+
+### Breaking Changes
+* **build** Changed compiling option to AOT (Ahead-of-time) compiling.
+
+### General
+* **styles** Styles are now in a .css-file rather than a .scss-file
+* **angular** Terra-Components is now a functioning angular-module and code autocompletion should work fine in your angular app
```

Comment:
Das gehört zum **build**

Reply:
Würde eher sowas schreiben wie "now complies to the [angular package format](https://docs.google.com/document/d/1CZC2rcpxffTDfRDs6p1cfbmKNLA6x5O-NtkJglDaBVs/preview)"